teams/Aragon One/2019.md

```diff
@@ -122,9 +113,50 @@ The Aragon Court is a service protocol designed to incentivize honest arbitratio
 
 ANT is intended to be used for governance and as the native currency for the network. In order to optimize ANT for these uses adjustments to ANT’s monetary policy may be necessary. This research would (1) explore the impact of creating a liquidity reserve and treasury bonds to tightly couple the networks assets with the valuation of ANT and (2) the feasibility of shifting the realization value appreciation to unit count rather than unit price using a scalar value token structure.
 
+### I09 – Caching on the Aragon client
+
+#### 🔜 Enhance aragonJS caching
+
+Make sure that aragonJS provides a much better caching mechanism, so the first load is slightly faster, and subsequent loads are much faster than they are now.
+
+#### 🔜 Experiment and implement centralized/decentralized hybrid for first-load
+
+Even enhancing aragonJS caching, the first load may still take a while. That's why it makes sense to experiment with centralized/decentralized hybrid solutions for it. For example, a trusted set of participants in the DAO could decide what the state was a at a given block, and then everyone would start loading from that. Then over time the client could sync locally and arrive to a fully decentralized/trustless state.
+
+### 🔜 I10 – Email notifications
+
+All actions on DAOs are irreversible and some of them may involve funds, so notifications become very important to the DAO's stakeholders.
+
+We will implement opt-in notifications using an email server and another server which will track all DAO-related events on Ethereum.
+
+### 🔜 I11 – Aragon client re-design and general enhancements
+
+We plan to re-design the Aragon client by creating a new Aragon Human Interface Guidelines that standardizes how to create Aragon apps.
+
+As part of this re-design, we will also re-design the following items, based on user feedback:
+
+- The Permissions app
+- The Voting app
+
+### 🔜 I12 – Support Aragon Funraising development
+
+We commit to supporting [Aragon Black](https://github.com/aragon/flock/blob/master/teams/Aragon%20Black/2019%20-%20Q3%20%26%20Q4.md) on their efforts to create the Aragon Fundraising app.
+
+This includes:
+
+- Designing the whole app
+- Doing code reviews
+- Assist in implementing a fundraising template on the Aragon client
+
+### 🔜 I13 – Launch Aragon Payroll
+
+Crypto-native payroll has sparked a lot of interest, and we are continuing Protofire's work on [Aragon Payroll](https://github.com/aragon/aragon-apps/tree/master/future-apps/payroll) to finish it and start using it with early testers.
+
+While we commit to getting it to a beta state, the maintainance of it will have to be carefully considered since Aragon One may not have the bandwidth to maintain it.
+
 ### aragonOS
 
-#### Global emergency failsafe
+#### 🔜 Global emergency failsafe
 
 Provide an opt-in failsafe for organizations and individual apps to safely freeze their state during times of emergency
 
```

Comment:
We can link to https://github.com/aragon/aragonOS/pull/518 instead of the issue.

Reply:
Solved, thanks!